OrbitGl/App.cpp

```diff
@@ -878,6 +936,31 @@ std::shared_ptr<Process> OrbitApp::FindProcessByPid(int32_t pid) {
   return it->second;
 }
 
+void OrbitApp::SelectFunction(const orbit_client_protos::FunctionInfo& func) {
+  uint64_t absolute_address = FunctionUtils::GetAbsoluteAddress(func);
+  LOG("Selected %s at 0x%" PRIx64 " (address_=0x%" PRIx64
+      ", load_bias_= 0x%" PRIx64 ", base_address=0x%" PRIx64 ")",
+      func.pretty_name(), absolute_address, func.address(), func.load_bias(),
+      func.module_base_address());
+  data_manager_->SelectFunction(absolute_address);
+}
+void OrbitApp::UnSelectFunction(const orbit_client_protos::FunctionInfo& func) {
```

Comment:
I would use `Unselect` (or honestly `Deselect`) instead of `UnSelect`, it's a single word. Here and for the other "unselect" methods in the pull request.

Reply:
changed to Deselect.